seed_data/lib.py

```diff
@@ -338,6 +337,7 @@ def set_course_run_current(course_run, enrollable_now=True,  # pylint: disable=t
     else:
         course_run.enrollment_start, course_run.enrollment_end = create_future_date_range()
     course_run.start_date = course_run.enrollment_start + timedelta(days=5)
+    course_run.end_date = course_run.start_date + timedelta(days=30)
```

Comment:
yes, the 3 conditionals above will always set a valid datetime for `enrollment_start`, so as a result `start_date` will always have a valid datetime (line 339)